apps/emqx_management/src/emqx_mgmt_api_trace.erl

```diff
@@ -633,8 +630,12 @@ read_file(Path, Offset, Bytes) ->
 parse_node(Query, Default) ->
     try
         case maps:find(<<"node">>, Query) of
-            error -> {ok, Default};
-            {ok, Node} -> {ok, binary_to_existing_atom(Node)}
+            error ->
+                {ok, Default};
+            {ok, NodeBin} ->
+                Node = binary_to_existing_atom(NodeBin),
+                true = lists:member(Node, mria_mnesia:running_nodes()),
```

Comment:
This'll turn into a 500 if the node is not part of the cluster or offline.  Maybe it could return an error that becomes 400?

Reply:
It's turned into `{error, not_found}` by try-catch and then later turned into `400`

Reply:
Maybe bad style, lazy me trying to save a few lines since we have the try-catch anyway. :/